src/lib/components/footer.svelte

```diff
@@ -4,13 +4,13 @@
       <a
         class="transition duration-200 hover:text-eddiehub-200 active:text-eddiehub-200"
         href="https://discord.com/invite/jZQs6Wu"
-        target="_blank">Discord</a
+        target="_blank"><i class="fa-brands fa-discord" /></a
```

Comment:
Because these links no longer contain any text they need an extra attribute to make them accessible to screen readers. Please add aria-label="Discord" and aria-label="Github" to the respective anchor tag.

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix